ydb/core/mind/hive/hive_impl.cpp

```diff
@@ -2771,13 +2804,12 @@ void THive::RequestPoolsInformation() {
         THolder<TEvBlobStorage::TEvControllerSelectGroups> ev = MakeHolder<TEvBlobStorage::TEvControllerSelectGroups>();
         NKikimrBlobStorage::TEvControllerSelectGroups& record = ev->Record;
         record.SetReturnAllMatchingGroups(true);
-        record.SetBlockUntilAllResourcesAreComplete(true);
```

Comment:
why?

Reply:
Because the call hanged forever in tests. Restored the option and tweaked the test instead.